charts/stash/templates/cleaner.yaml

```diff
@@ -24,7 +24,13 @@ spec:
            kubectl delete validatingwebhookconfigurations admission.stash.appscode.com || true; \
            kubectl delete mutatingwebhookconfiguration admission.stash.appscode.com || true; \
            kubectl delete apiservice v1alpha1.admission.stash.appscode.com || true; \
+           kubectl delete apiservice v1alpha1.appcatalog.appscode.com || true; \
+           kubectl delete apiservice v1alpha1.stash.appscode.com || true; \
+           kubectl delete apiservice v1beta1.stash.appscode.com || true; \
```

Comment:
These are created as a result of Stash CRDs. We should not delete these directly. Also, we don't delete stash crd from cluster and I don't think we should delete them.